### PR TITLE
Add "Keyboard navigation" system preference

### PR DESCRIPTION
**Pull request by TrevorBurnham**
This PR adds a page for the "Keyboard navigation" system preference, which allows users to use Tab and Shift+Tab to navigate in all system dialogs.

### CODE REVIEW

**Comment by yannbertrand** [comment]
Didn't know about that one, thanks for the discovery!

Could you add some example usages?

[That article](https://www.tempertemper.net/blog/using-the-keyboard-to-navigate-on-macos) looks interesting as well.

File: docs/keyboard/applekeyboarduimode.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Reset to default value
+
+```bash
+defaults delete NSGlobalDomain AppleKeyboardUIMode
+```
+
+## Set value from UI
+
+1. Open the Keyboard preference pane in System Preferences
```
Comment (yannbertrand) [inline docs/keyboard/applekeyboarduimode.md:62]:
```suggestion
1. <a href="x-apple.systempreferences:com.apple.preference.keyboard">Open the Keyboard preference pane in System Preferences</a>
```

**Comment by TrevorBurnham** [comment]
I've updated the PR with screenshots to demonstrate a classic use case: Navigating a "Do you want to save?" dialog. See e.g. this popular Ask Different question: https://apple.stackexchange.com/questions/55292/how-do-i-negotiate-dialogue-boxes-using-the-keyboard-only

Let me know if there's anything I can do to make the functionality clearer. I'd be happy to add an annotation to the "enabled" screenshot, but I don't see any examples of that in the existing screenshots.

**Comment by yannbertrand** [comment]
> Let me know if there's anything I can do to make the functionality clearer. I'd be happy to add an annotation to the "enabled" screenshot, but I don't see any examples of that in the existing screenshots.

Yea it's about limiting the screenshots. When updating to a new macOS version I try to recapture all existing screenshots but it can take a lot of time.

**Comment by yannbertrand** [comment]
This is great! Thank you

**Pull request merged**